client/components/header.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+class Header extends React.Component {
+  render() {
+    return (
+      <header>
+        <div className="collapse bg-dark" id="navbarHeader">
+          <div className="container">
+            <div className="row">
+              <div className="col-sm-8 col-md-7 py-4">
+                <h4 className="text-white">About</h4>
```

Comment:
I don't think there's any of the about info in this part of the project, this should just be the header with the $ icon and Wicked Sales